ietf-layer0-types-ext.yang

```diff
@@ -277,21 +287,21 @@ module ietf-layer0-types-ext {
                    mode.";
                 leaf-list supported-application-codes {
                   type leafref {
-                    path "../../mode-id";
+                    path "../../../mode-id";
                   }
-                  must "../../../supported-mode[mode-id=current()]/"
+                  must "../../../../supported-mode[mode-id=current()]/"
                      + "standard-mode" {
                     description
                       "The pointer is only for application codes
-                       supported by transceiver.";
+                       supported by transceiver."; 
                   }
                   description
                     "List of pointers to the application codes
                      supported by the transceiver's explicit mode.";
                 }
                 leaf-list supported-organizational-modes {
                   type leafref {
-                    path "../../mode-id";
+                    path "../../../mode-id";
                   }
                   must "../../../supported-mode[mode-id=current()]/"
                      + "operational-mode" {

```

Comment:
It seems that also this must statement is not correct

```suggestion
                  must "../../../../"
                     + "supported-mode[mode-id=current()]/"
                     + "organizational-mode" {
```

Co-authored-by: sergiobelotti <sergio.belotti@nokia.com>

Reply:
Addressed with https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/11/commits/9e2c4758a623ed7879d84d055d58807d3df2bc51